app/keepers.go

```diff
@@ -61,68 +61,65 @@ import (
-	porttypes "github.com/cosmos/ibc-go/v2/modules/core/05-port/types"
 
-	// IBC: Inter-blockchain communication
+	// IBC: Inter-blockchain communication.
 	ibcclient "github.com/cosmos/ibc-go/v2/modules/core/02-client"
+	ibcclienttypes "github.com/cosmos/ibc-go/v2/modules/core/02-client/types"
+	porttypes "github.com/cosmos/ibc-go/v2/modules/core/05-port/types"
 	ibchost "github.com/cosmos/ibc-go/v2/modules/core/24-host"
 	ibckeeper "github.com/cosmos/ibc-go/v2/modules/core/keeper"
 
-	// Upgrades from earlier versions of Osmosis
+	// Modules related to bech32-ibc, which allows new ibc funcationality based on the bech32 prefix of addresses.
+	"github.com/osmosis-labs/bech32-ibc/x/bech32ibc"
+	bech32ibckeeper "github.com/osmosis-labs/bech32-ibc/x/bech32ibc/keeper"
+	bech32ibctypes "github.com/osmosis-labs/bech32-ibc/x/bech32ibc/types"
+	bech32ics20keeper "github.com/osmosis-labs/bech32-ibc/x/bech32ics20/keeper"
+
+	// Upgrades from earlier versions of Osmosis.
 	_ "github.com/osmosis-labs/osmosis/v7/client/docs/statik"
 
-	// Modules that live in the Osmosis repository and are specific to Osmosis
+	// Modules that live in the Osmosis repository and are specific to Osmosis.
 	claimkeeper "github.com/osmosis-labs/osmosis/v7/x/claim/keeper"
 	claimtypes "github.com/osmosis-labs/osmosis/v7/x/claim/types"
-	"github.com/osmosis-labs/osmosis/v7/x/superfluid"
-	"github.com/osmosis-labs/osmosis/v7/x/txfees"
 
-	// Epochs: gives Osmosis a sense of "clock time" so that events can be based on days instead of "number of blocks"
+	// Epochs: gives Osmosis a sense of "clock time" so that events can be based on days instead of "number of blocks".
 	epochskeeper "github.com/osmosis-labs/osmosis/v7/x/epochs/keeper"
 	epochstypes "github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 
-	// Generalized Automated Market Maker
+	// Generalized Automated Market Maker.
 	gammkeeper "github.com/osmosis-labs/osmosis/v7/x/gamm/keeper"
 	gammtypes "github.com/osmosis-labs/osmosis/v7/x/gamm/types"
 
-	// Incentives: Allows Osmosis and foriegn chain communities to incentivize users to provide liquidity
+	// Incentives: Allows Osmosis and foreign chain communities to incentivize users to provide liquidity.
 	incentiveskeeper "github.com/osmosis-labs/osmosis/v7/x/incentives/keeper"
 	incentivestypes "github.com/osmosis-labs/osmosis/v7/x/incentives/types"
 
-	// Lockup: allows tokens to be locked (made non-transferrable)
+	// Lockup: allows tokens to be locked (made non-transferrable).
 	lockupkeeper "github.com/osmosis-labs/osmosis/v7/x/lockup/keeper"
 	lockuptypes "github.com/osmosis-labs/osmosis/v7/x/lockup/types"
 
-	// Mint: Our modified version of github.com/cosmos/cosmos-sdk/x/mint
+	// Mint: Our modified version of github.com/cosmos/cosmos-sdk/x/mint.
 	mintkeeper "github.com/osmosis-labs/osmosis/v7/x/mint/keeper"
 	minttypes "github.com/osmosis-labs/osmosis/v7/x/mint/types"
 
-	// Pool incentives:
+	// Pool incentives:.
```

Comment:
Did we get to the bottom of why these priods after colons / can we fix them / remove alot of the excess periods?

I think we  should disable whatever linter caused this

Reply:
like if we grep for `:.` and just revert those back to `:`